tools/pipelines/templates/build-npm-package.yml

```diff
@@ -216,11 +216,6 @@ extends:
               - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
                 - name: targetBranchName
                   value: $(System.PullRequest.TargetBranch)
-              # Absolute path to the folder that contains the source code for the telemetry-generator package, which is
-              # used in a few places in the pipeline to push custom telemetry to Kusto.
-              - name: absolutePathToTelemetryGenerator
-                value: $(Build.SourcesDirectory)/tools/telemetry-generator
-                readonly: true
```

Comment:
Moved the variable to the new `include-vars-telemetry-generator.yml` template, which is in turn included by `include-vars.yml`. Similar snippets were removed in other yml files for the same reason.